EnglishFilter/sections/specific.txt

```diff
@@ -261,6 +261,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+claimfreecoins.io##div[id^="-adspace"]
```

Comment:
It doesn't apply on my end. Try `claimfreecoins.io##div[id$="-adspace"]`

Reply:
Changed.

Reply:
Ok, thanks.

Reply:
You're welcome.